src/main/java/com/github/cfogrady/dim/modifier/SpriteImageTranslator.java

```diff
@@ -66,6 +68,16 @@ public ObservableList<ImageIntComboBox.ImageIntPair> createImageValuePairs(List<
         }
         return items;
     }
+    public ObservableList<ImageIntListView.ImageIntPair> createImageValuePairsForListView(List<SpriteData.Sprite> sprites) {

```

Comment:
As per below, I'd advocate for making ImageIntPair independent of the ImageIntListView and ImageIntComboBox classes. Then both those classes can refer to the same ImageIntPair, and we can just use the existing method instead of duplicating a new one.